e2e/app.spec.ts

```diff
@@ -34,20 +35,23 @@ const ONEX_BBOX_HEIGHT = distance(point(ONEX_BBOX.southWest), point([ ONEX_BBOX.
 
 describe('App', function() {
 
-  let locations;
+  let locations, actions;
   let mapPage: MapPageObject;
   let actionsListPage: ActionsListPageObject;
+  let actionPage: ActionPageObject;
 
   beforeEach(async function() {
     mapPage = new MapPageObject('map-page');
     actionsListPage = new ActionsListPageObject('actions-list-page');
+    actionPage = new ActionPageObject('action-page');
```

Comment:
I'm only noticing this now, and I may well have started it, but I don't think we should have to specify the selector here in each test. It should be a default value in those classes' constructors.

Reply:
It is now ;)
I've come to the same conclusion in the last couple of PR, and made the change there (among other changes)